Tools/scripts/deepfreeze.py

```diff
@@ -313,6 +314,8 @@ def _generate_int_for_bits(self, name: str, i: int, digit: int) -> None:
                 self.write(f".ob_digit = {{ {ds} }},")
 
     def generate_int(self, name: str, i: int) -> str:
+        if -5 <= i <= 256:
```

Comment:
(I wish there was a Python API to find out these boundaries. But there isn't one, so I'm okay with writing it this way.)

Reply:
Yes, there isn't any API for this so just had to look up the values in headers.